app/decorators/service_provider_session.rb

```diff
@@ -11,8 +11,8 @@ def initialize(sp:, view_context:, sp_session:, service_provider_request:)
     @service_provider_request = service_provider_request
   end
 
-  def remember_device_default
-    sp_aal < 2
+  def remember_device_default(authorization_context)
```

Comment:
If we made the authentication context an argument to the initializer we could pass it in in the application controller and not need to worry about passing it into all of the methods that will need to consume it.